src/ui_basic/window.cc

```diff
@@ -64,12 +70,6 @@ constexpr int16_t kVerticalBorderThingyHeight = 20;
 constexpr int16_t kVerticalBorderMiddleLength =
    (kVerticalBorderTotalLength - 2 * kVerticalBorderThingyHeight);
 
-static const std::string kWindowImageClose = kTemplateDir + "wui/window_close.png";
-static const std::string kWindowImagePinned = kTemplateDir + "wui/window_unpin.png";
-static const std::string kWindowImageUnpinned = kTemplateDir + "wui/window_pin.png";
-static const std::string kWindowImageMinimize = kTemplateDir + "wui/window_minimize.png";
-static const std::string kWindowImageMaximize = kTemplateDir + "wui/window_maximize.png";
```

Comment:
Reintroduce them as `constexpr const char* const`. Some of these constants are used in multiple places so they should be typed only once.

Reply:
The only way to to that is to not use `kTemplateDir`, because I have never found a way to directly concatenate 2 `const char* const` variables. So, we'd type `kTemplateDir` 4 times instead. If we don't come up with anything else, we could turn them into macros. In the long run, I want to make the template dir configurable anyway, so I'd be fine with a macro here if you agree.